src/metrics.rs

```diff
@@ -2032,73 +2059,102 @@ impl GooseMetrics {
     /// This function is invoked by [`GooseMetrics::print()`].
     pub(crate) fn fmt_overview(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
         // Only display overview in the final metrics.
-        if !self.final_metrics || self.starting.is_none() {
+        if !self.final_metrics || self.history.is_empty() {
             return Ok(());
         }
 
-        // Calculations necessary for overview table.
-        let starting = self.starting.unwrap();
-        let starting_time = starting.format("%Y-%m-%d %H:%M:%S").to_string();
-        let started = if self.started.is_some() {
-            self.started.unwrap()
-        } else {
-            self.stopping.unwrap()
-        };
-        let (starting_seconds, starting_minutes, starting_hours) =
-            self.get_seconds_minutes_hours(&starting, &started);
-        let start_time = started.format("%Y-%m-%d %H:%M:%S").to_string();
-        let stopping = self.stopping.unwrap();
-        let (running_seconds, running_minutes, running_hours) =
-            self.get_seconds_minutes_hours(&started, &stopping);
-        let stopping_time = stopping.format("%Y-%m-%d %H:%M:%S").to_string();
-        let stopped = self.stopped.unwrap();
-        let stopped_time = stopped.format("%Y-%m-%d %H:%M:%S").to_string();
-        let (stopping_seconds, stopping_minutes, stopping_hours) =
-            self.get_seconds_minutes_hours(&stopping, &stopped);
+        writeln!(
+            fmt,
+            "\n === OVERVIEW ===\n ------------------------------------------------------------------------------"
+        )?;
 
+        writeln!(
+            fmt,
+            " {:<12} {:<19} {:<17} {:<10} Users",
+            "Action", "Started", "Stopped", "Elapsed",
+        )?;
```

Comment:
I am wondering if this output really belongs in this file. Are these metrics or something else? 

`metrics.rs` is already a huge file and we might want to consider splitting it at some point? Maybe this is a good time to start this discussion (actually doing it is out of scope for this PR of course).

Reply:
This is the ASCII display of the metrics. I suppose it could be moved into report.rs if that's a better fit?